stable/postgresql/templates/_helpers.tpl

```diff
@@ -14,3 +14,14 @@ We truncate at 63 chars because some Kubernetes name fields are limited to this
 {{- $name := default .Chart.Name .Values.nameOverride -}}
 {{- printf "%s-%s" .Release.Name $name | trunc 63 | trimSuffix "-" -}}
 {{- end -}}
+
```

Comment:
I'm not sure creating this 7 line helper function is necessary in this case. It is only used once and the if statement can be inlined in the deployment in one line. (E.g. https://github.com/kubernetes/charts/blob/0e646ace488694573d921df8e4aa5ebb18dcca25/stable/prometheus/templates/server-deployment.yaml#L71). IMO this indirection costs more then it provides.